src/sentry/lang/javascript/processor.py

```diff
@@ -880,22 +875,34 @@ def __init__(self, *args, **kwargs):
 
         self.organization = organization
         self.max_fetches = MAX_RESOURCE_FETCHES
-        self.allow_scraping = organization.get_option(
-            "sentry:scrape_javascript", True
-        ) is not False and self.project.get_option("sentry:scrape_javascript", True)
 
         self.fetch_count = 0
         self.sourcemaps_touched = set()
 
-        # cache holding mangled code, original code, and errors associated with
-        # each abs_path in the stacktrace
-        self.cache = SourceCache()
+        # All the following dictionaries have been defined top level for simplicity reasons. Because this code will
+        # be ported to Symbolicator we wanted to keep it as simple and explicit as possible. This comment also
+        # applies to the many repetitions across the code.
+
+        # Cache holding the results of the fetching by url.
+        self.fetch_by_url_sourceviews = {}
+        self.fetch_by_url_errors = {}
```

Comment:
is this the only place that stores (cached) download errors?

Reply:
Yep, it is.